exercises/practice/atbash-cipher/.meta/example.cairo

```diff
@@ -0,0 +1,78 @@
+pub fn encode(phrase: ByteArray) -> ByteArray {
+    let mut result = "";
+
+    let mut i = 0;
+    let mut chunk_size = 0;
+    while i < phrase.len() {
+        let chr = phrase[i];
+
+        if is_whitespace(chr) || is_punctuation(chr) {
+            i += 1;
```

Comment:
nit: you can increment `i` above the if-statement, which would also remove the need to increment it at the bottom of the loop ([line 26](https://github.com/exercism/cairo/pull/274/files#diff-2a9522c77c2a98f38d12114741a42f53d3b85b61e04885c708df2c5263464aadR26)).

Reply:
Fair. I thought about that, but it felt weird to increment my counter before I did anything. It'd make the code more clear though.